documentation/source/newsfragments/2022.change.rst

```diff
@@ -0,0 +1 @@
+Updated :class:`~cocotb.drivers.Driver`, :class:`~cocotb.monitors.Monitor`, and all subclasses to use the :keyword:`async`/:keyword:`await` syntax, instead of the :keyword:`yield` syntax.
```

Comment:
Maybe
```suggestion
Updated :class:`~cocotb.drivers.Driver`, :class:`~cocotb.monitors.Monitor`, and all their subclasses to use the :keyword:`async`/:keyword:`await` syntax, instead of the :keyword:`yield` syntax.
```
I would have left out the last two commas but maybe that's overcompensating the tendency in German to have lots of commas.

Reply:
Also removed the last comma. It's unnecessary.